### ISSUE TITLE
FR does not update after entering it

### ISSUE THREAD

**Issue by mpawlowski85** [unicef/etools-issues#440]
Workspace: Iraq
Record link: https://etools.unicef.org/pmp/interventions/149/details
Environment: PROD

Steps to reproduce: 
1. Go to the record link
2. Try to enter FR 0400050080 into the "Review and Sign Tab" 
3.

**Comment by anusenan** [comment]
@mpawlowski85  is this desc complete? What is the error message you get? Can you please share a screenshot if available?

**Comment by mpawlowski85** [comment]
Yes - the description is complete. You can just go to the record itself, but I am including the screenshot for convenience. There is no error message. 

![image](https://user-images.githubusercontent.com/26120371/27104891-87d8c798-505b-11e7-9f68-60965eae3f58.png)

**Comment by mpawlowski85** [comment]
On further investigation, it seems that the problem stems from the failure in FR and FC synchronizer. The failure occurred for multiple countries.

**Comment by anusenan** [comment]
@ntrncic similar to 440

**Comment by ntrncic** [comment]
@janpanchalingam1 @mpawlowski85 this exact FR is already used by another PD, this is why you cannot add it. I've tried other FRs and the work in Demo.

**Comment by janpanchalingam1** [comment]
@ntrncic yes, that makes sense. Although, now we are unable to edit/add another FR:
![image](https://user-images.githubusercontent.com/26144740/28831967-404b016a-76a9-11e7-821f-02c0d0f4d100.png)

**Comment by ntrncic** [comment]
@janpanchalingam1 from our conversations with Marcin it is One FR per One PD.

**Comment by janpanchalingam1** [comment]
passed testing
